pkg/results/reader.go

```diff
@@ -147,6 +147,21 @@ func (r *Reader) WalkFiles(walkfn filepath.WalkFunc) error {
 
 // Functions to be used within a walkfn
 
+// ExtractBytes pulls out bytes into a buffer for any path matching file.
+func ExtractBytes(file string, path string, info os.FileInfo, buf *bytes.Buffer) error {
```

Comment:
try this

if file != path {
  return nil
}
reader, ok := info.Sys().(io.Reader)
 if !ok {
 		return errors.New("info.Sys() is not a reader")
 }
 _, err := buf.ReadFrom(reader)
return err